wpimath/src/main/native/include/frc/controller/BangBangController.h

```diff
@@ -40,7 +42,13 @@ class WPILIB_DLLEXPORT BangBangController
    */
   constexpr explicit BangBangController(
       double tolerance = std::numeric_limits<double>::infinity())
-      : m_tolerance(tolerance) {}
+      : m_tolerance(tolerance) {
+    if (!std::is_constant_evaluated()) {
```

Comment:
Won't this not report any instances created in a constexpr context? How significant of a miss is that?

Reply:
We're mainly interested in the ones used at runtime.

Reply:
This is the same as PIDController